src/main/java/com/powsybl/openloadflow/network/LfNetworkParameters.java

```diff
@@ -597,6 +602,22 @@ public LfNetworkParameters setAreaInterchangeControlAreaType(String areaIntercha
         return this;
     }
 
+    public int getMaxVoltageRemoteControlDistance() {
+        return maxVoltageRemoteControlDistance;
+    }
+
+    public static int checkMaxVoltageRemoteControlDistance(int maxVoltageRemoteControlDistance) {
+        if (maxVoltageRemoteControlDistance < 0) {
+            throw new IllegalArgumentException("Invalid value for parameter maxVoltageRemoteControlDistance: " + maxVoltageRemoteControlDistance);
+        }
```

Comment:
The message is not clear. It should explain the rules (>=0 ?)

But do we need this ? Can't we say that <=0 means no check ?

Reply:
Yes this is simpler to say that <= 0 means no check